*: update the row count display of probe side of IndexJoin and Apply (#10864)
pingcap/docs: explain-overview.md

* `access object` shows the table, partition and index that is being accessed. The parts of the index are also shown, as in the case above that the column `a` from the index was used. This can be useful in cases where you have composite indexes.
* `operator info` shows additional details about the access. See [Operator info overview](#operator-info-overview) for additional details.

> **Note:**
>
> In the execution plan, for all Probe side operators of `IndexJoin` and `Apply`, the meaning of `estRows` from v6.4.0 is different that before v6.4.0.

> > Before v6.4.0, `estRows` means the estimated row count expected to be processed by the Probe side operators for each row from the Build side operator. From v6.4.0, `estRows` means the estimated **total row count** expected to be processed by the Probe side operators. The actual row count displayed (the `actRows` column) in the `EXPLAIN ANALYZE` statement means the total row count, so the meanings of `estRows` and `actRows` for those operatos are consistent from v6.4.0.

Reviewer: ```suggestion
> Before v6.4.0, `estRows` means the number of estimated rows to be processed by the probe side operators for each row from the build side operators. Since v6.4.0, `estRows` means the **total number** of estimated rows to be processed by the probe side operators. The actual number of rows displayed (indicated by the `actRows` column) in the result of `EXPLAIN ANALYZE` means the total row count, so since v6.4.0 the meanings of `estRows` and `actRows` for those `IndexJoin` and `Apply` operators are consistent.
```

Author: The last sentence is inaccurate. It's not "IndexJoin and Apply" but "the probe side children of IndexJoin and Apply".

So I wonder if "those operators" is enough. Or we should add "the probe side children of".

> > Before v6.4.0, `estRows` means the number of estimated rows to be processed by the probe side operators for each row from the build side operators. Since v6.4.0, `estRows` means the **total number** of estimated rows to be processed by the probe side operators. The actual number of rows displayed (indicated by the `actRows` column) in the result of `EXPLAIN ANALYZE` means the total row count, so since v6.4.0 the meanings of `estRows` and `actRows` for the probe side child nodes of `IndexJoin` and `Apply` operators are consistent.

Reviewer: ```suggestion
>
> Before v6.4.0, `estRows` means the number of estimated rows to be processed by the probe side operators for each row from the build side operators. Since v6.4.0, `estRows` means the **total number** of estimated rows to be processed by the probe side operators. The actual number of rows displayed (indicated by the `actRows` column) in the result of `EXPLAIN ANALYZE` means the total row count, so since v6.4.0 the meanings of `estRows` and `actRows` for the probe side child nodes of `IndexJoin` and `Apply` operators are consistent.
```

> |   ├─TableReader_16(Build)       | 10000.00 | root      |                       | data:TableFullScan_15                                                        |
> |   │ └─TableFullScan_15          | 10000.00 | cop[tikv] | table:t1              | keep order:false, stats:pseudo                                               |
> |   └─Limit_17(Probe)             | 1.00     | root      |                       | offset:0, count:1                                                            |
> |     └─IndexReader_21            | 1.00     | root      |                       | index:Limit_20                                                               |
> |       └─Limit_20                | 1.00     | cop[tikv] |                       | offset:0, count:1                                                            |
> |         └─IndexRangeScan_19     | 1.00     | cop[tikv] | table:t2, index:ia(a) | range: decided by [eq(test.t2.a, test.t1.b)], keep order:false, stats:pseudo |
> +---------------------------------+----------+-----------+-----------------------+------------------------------------------------------------------------------+
> 

> > -- From v6.4.0:

Reviewer: ```suggestion
> -- Since v6.4.0:
```

>

> > -- You can find the difference between `estRows` of `IndexLookUp_11`、`Selection_10`、`IndexRangeScan_8` and `TableRowIDScan_9` and that before v6.4.0.

Reviewer: ```suggestion
> -- You can find that the `estRows` column values for `IndexLookUp_11`, `Selection_10`, `IndexRangeScan_8`, and `TableRowIDScan_9` since v6.4.0 is different from that before v6.4.0.
```

> | │   └─TableFullScan_22          | 10000.00 | cop[tikv] | table:t1              | keep order:false, stats:pseudo                                                                                  |
> | └─IndexLookUp_11(Probe)         | 12487.50 | root      |                       |                                                                                                                 |
> |   ├─Selection_10(Build)         | 12487.50 | cop[tikv] |                       | not(isnull(test.t2.a))                                                                                          |
> |   │ └─IndexRangeScan_8          | 12500.00 | cop[tikv] | table:t2, index:ia(a) | range: decided by [eq(test.t2.a, test.t1.a)], keep order:false, stats:pseudo                                    |
> |   └─TableRowIDScan_9(Probe)     | 12487.50 | cop[tikv] | table:t2              | keep order:false, stats:pseudo                                                                                  |
> +---------------------------------+----------+-----------+-----------------------+-----------------------------------------------------------------------------------------------------------------+
>

> > -- You can find the difference between `estRows` of `Limit_17`、`IndexReader_21`、`Limit_20` and `IndexRangeScan_19` and that before v6.4.0.

Reviewer: ```suggestion
> -- You can find that the `estRows` column values for `Limit_17`, `IndexReader_21`, `Limit_20`, and `IndexRangeScan_19` since v6.4.0 is different from that before v6.4.0.
```
